### ISSUE TITLE
Required Indian GST implementation for all products

### ISSUE THREAD

**Issue by unaveen** [aimeos/ai-admin-jqadm#95]
We need a feature to calculate Indian GST Tax for all products

**Comment by aimeos** [comment]
Can you please explain in more detail?
How is it calculated and what must be done additionally to the current tax calculation?

**Comment by aimeos** [comment]
You can now add several tax rates in the admin interface for each price:
![admin-price-taxrates](https://user-images.githubusercontent.com/8647429/65376325-d2238f00-dc9e-11e9-91ad-991e1b7e5964.png)

To configure them in your config file:
```
'admin' => [
	'tax' => [
		'gst' => '8.50',
		'pst' => '7.25',
	]
]
```

Tax calculation using several tax rates in the frontend is already available since version 2019.07
